src/layout/header/index.js

```diff
@@ -1,205 +1,145 @@
-            <a
-              href="https://pantheon.io/register"
-              target="_blank"
-              rel="noopener"
-              className="ctaBtn cta cta-yellow docs-cta"
-            >
-              Get Free Account
-            </a>
-          </li>
-          <li className="">
-            <a
-              href="https://dashboard.pantheon.io"
-              target="_blank"
-              rel="noopener"
-            >
-              Log In
-            </a>
-          </li>
-          <li className="">
-            <a
-              href="https://dashboard.pantheon.io/#support"
-              target="_blank"
-              rel="noopener"
-            >
-              Support
-            </a>
-          </li>
-          <li className="dropdown">
-            <a
-              href="/"
-              className="dropdown-toggle active-trail"
-              role="button"
-              aria-haspopup="true"
-              style={{ paddingLeft: "15px", paddingRight: "25px" }}
-            >
-              Documentation
-            </a>
+import {
+  Container,
+  InputText,
+  Navbar,
+  NavMenu,
+} from "@pantheon-systems/pds-toolkit-react"
 
-            <ul id="dropmenu" className="dropdown-menu">
-              <div className="row sub-menu-top">
-                <div className="col-xs-6">
-                  <li>
-                      <Link id="get-started" to="/get-started">
-                      Get Started
-                      </Link>
-                  </li>
-                  <li>
-                    <Link id="develop" to="/develop">
-                      Develop
-                    </Link>
-                  </li>
-                  <li>
-                    <Link id="go-live" to="/go-live">
-                      Go Live
-                    </Link>
-                  </li>
-                  <li>
-                    <Link id="platform" to="/platform">
-                      Explore Platform Architecture
-                    </Link>
-                  </li>
-                </div>
+import "./style.css"
 
-                <div className="col-xs-6">
-                  <li>
-                    <Link id="automate" to="/automate">
-                      Automate & Integrate
-                    </Link>
-                  </li>
-                  <li>
-                    <Link id="performance" to="/performance">
-                      Optimize Performance
-                    </Link>
-                  </li>
-                  <li>
-                    <Link id="manage" to="/manage">
-                      Manage Teams & Organizations
-                    </Link>
-                  </li>
-                  <li>
-                    <Link id="troubleshoot" to="/troubleshoot">
-                      Troubleshoot
-                    </Link>
-                  </li>
-                </div>
-              </div>
+// Links for NavMenu component.
+const mainNavigationLinks = [
+  {
+    linkContent: (
```

Comment:
It strikes me as a little odd to have these 4 links in the top left of the header with differing behaviors.

<img width="1674" alt="Screenshot 2023-09-28 at 2 37 08 PM" src="https://github.com/pantheon-systems/documentation/assets/211029/199bc87a-1278-4142-8464-2c3e62cd6192">


* The logo is like a home button next to Docs Home. But it goes to the top level `pantheon.io`. I think that's a little confusing but maybe fine.
* "Docs Home" next to a drop down labeled "Documentation" feels duplicative in a different way.
* Then "Support" goes to `dashboard.pantheon.io`. Which is the subdomain used by the links on the right of the header nav.

I don't want to hold up this PR unnecessarily. I just see opportunity for further improvement. Perhaps a small change that would alleviate my main concern would be changing "Documentation" to "Topics" or "Docs Topics" since it is a drop down of Topics

Reply:
That's fine. I can certainly change it to "Topics".  

This exercise is mainly to theme what was already there and try to improve if possible. The main reason for adding docs home is that on the current site, there is no way to get to the homepage from the navbar. Would you rather the Pantheon logo go to docs home?

Reply:
> The main reason for adding docs home is that on the current site, there is no way to get to the homepage from the navbar.

Looks like that on the current design, the "Documentation" link that triggers the drop down nav is itself a link to the docs home page. I think I'd prefer to keep that behavior and then drop the separate "Docs Home" link

Reply:
The menu component is not currently built to act that way for accessibility best practices. Top-level links are buttons that trigger the menus and not links themselves. See example: https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/

What if we just put the "Docs home" link as the top item in the dropdown for now — then after the IA recommendations are in, we can reevaluate if we need a different component?

Reply:
Also, do you want to move support to the right side?

Reply:
> Also, do you want to move support to the right side?

I don't have a strong opinion there. Let's not block this PR on it.